### PR TITLE
Adding Floyd Warshall Algorithm in C

### PR DESCRIPTION
**Pull request by p-soham11**
Fixes #2205 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Adding Floyd Warshall Algorithm in C

#### Languages Used:
- C

#### Files Added:
- Floyd_Warshall_Algorithm.c

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 1]
```diff
@@ -0,0 +1,100 @@
+//Floyd Warshall Algorithm using C
+
+#include<stdio.h>
+
+//Vertices count in the Graph
+#define A 4
+
+//Defining inf(infinite) as a large value.
+//It's used for non-connected vertices
+#define inf 999999
```
Comment (asha15) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:10]: define this in upper case letters

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            else
+                printf ("%7d", dist[i][j]);
+        }
+        printf("\n");
+    }
+}
+
+int main()
+{
+
```
Comment (asha15) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:66]: remove this line

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    int graph[A][A] = { {0,   9,  inf, 8},
```
Comment (asha15) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:67]: remove the space after the first brace

Comment (asha15) [reply Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:67]: keep only one space after comma

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                        {inf, 0,   6, inf},
+                        {inf, inf, 0,   5},
+                        {inf, inf, inf, 0}
+                      };
+
+    //Calling function to execute Floyd Warshall Algorithm
+    Floyd_Warshall(graph);
+    return 0;
+}
+
```
Comment (asha15) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:77]: remove this line

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+{
+
+    int graph[A][A] = { {0,   9,  inf, 8},
+                        {inf, 0,   6, inf},
+                        {inf, inf, 0,   5},
+                        {inf, inf, inf, 0}
+                      };
+
+    //Calling function to execute Floyd Warshall Algorithm
+    Floyd_Warshall(graph);
```
Comment (asha15) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:74]: define the function name in lower class letters

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+#define inf 999999
+
+//Declaring a function to print the solution matrix
+void solution(int dist[][A]);          
+
+//Function to solve ALL-PAIR SHORTEST PATH problem
+//using Floyd Warshall Algorithm
+void Floyd_Warshall (int graph[][A])
+{
+
```
Comment (asha15) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:19]: remove this line

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+      9 |          |
+        |          | 5
+       \|/         |
+       (1)------->(2)
+            6
+
+    OUTPUT :
+
+    The following matrix shows the shortest distance between
+    every pair of vertices:
```
Comment (asha15) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:94]: remove the spaces at the beginning

**Comment by p-soham11** [comment]
Ma'am, I have resolved all the proposed changes.

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+//Defining INF(infinite) as a large value.
+//It's used for non-connected vertices
+#define INF 999999
+
+//Declaring a function to print the solution matrix
+void solution(int dist[][A]);          
+
+//Function to solve ALL-PAIR SHORTEST PATH problem
+//using Floyd Warshall Algorithm
+void floyd_warshall (int graph[][A])
```
Comment (asha15) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:17]: please remove the space before open parenthesis

**Pull request closed without merging**